lib/deploy/utils/index.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path')
+const fs = require('fs-promise')
+
+const checkBuildFiles = (dir) => (
+  Promise.all([
+    checkFileExists(dir, 'index.html'),
```

Comment:
Now, this will be affected by the `--output-to-base` option. There might not be an index.html file because there will be a `base-path/index.html` file.